nxp/imx8qm-mek/overlay.nix

```diff
@@ -0,0 +1,3 @@
+final: _prev: {
+  inherit ( final.callPackage ./bsp/u-boot/imx8/imx-uboot.nix { pkgs = final; targetBoard = "imx8qm"; }) ubootImx8 imx-firmware;
```

Comment:
Is `./bsp/u-boot/imx8/imx-uboot.nix ` correct path?

Reply:
Hi, sorry for this. I'll fix paths and improve README